site/content/docs/5.0/components/alerts.md

```diff
@@ -128,7 +128,7 @@ This makes an alert listen for click events on descendant elements which have th
         <code>dispose</code>
       </td>
       <td>
-        Destroys an element's alert.
+        Destroys an element's alert. (Removes stored data on the DOM element and unset the alert's attributes.)
```

Comment:
`unsets`. There's also an unneeded period I believe inside the parens.

Reply:
@XhmikosR Updated.